pynest/nest/__init__.py

```diff
@@ -178,8 +173,12 @@ def test():
 
     hl_api.set_debug(debug)
 
+from .pynestkernel import *
+from lib.hl_api_helper import *
+
+for name in os.listdir(os.path.join(os.path.dirname(__file__), "lib")):
+    if name.endswith(".py") and not name.startswith('__'):
+        exec("from lib.{0} import *".format(name[:-3]))
```

Comment:
@heplesser, you're right. This code is indeed a bit cryptic (and the result of endless hours of failure) and deserves a comment that explains it. I've added one in 5afad17